storage/filesystem/storage_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"testing"
 
 	"gopkg.in/src-d/go-git.v4/storage/test"
-	"gopkg.in/src-d/go-git.v4/utils/fs/os"
+	"srcd.works/go-billy.v0/os"
```

Comment:
I've gone through all the changed files and added it to a third group. When needed, I've grouped other imports as well.

Thank you for the review!